dev/ts/component/SvgExporter.ts

```diff
@@ -162,7 +162,7 @@ class CanvasForExport extends mxSvgCanvas2D {
 
   private computeTruncatedText(content: string, w: number): string {
     // Assumes a max character width of 0.5em
-    if (content == null || this.state.fontSize <= 0) {
+    if (content == undefined || this.state.fontSize <= 0) {
```

Comment:
**nitpick:** mxGraph code, please revert. In addition, the rule requiring to use undefined instead of null has been disabled.